b2b_ecommerce/models.py

```diff
@@ -153,6 +154,24 @@ class B2BOrder(OrderAbstract, AuditableModel):
         on_delete=models.PROTECT,
     )
 
+    # These represent the tax collected for the entire order.
+    tax_country_code = models.CharField(max_length=2, blank=True, null=True)
+    tax_rate = models.DecimalField(
+        max_digits=6, decimal_places=4, null=True, blank=True, default=0
+    )
+    tax_rate_name = models.CharField(max_length=100, null=True, default="VAT")
+
+    # These represent the customer's tax identifier for the order
+    customer_tax_identifier = models.TextField(default="")
+    customer_tax_identifier_name = models.CharField(
+        max_length=100, null=True, default=""
+    )
+
+    # This is the FK to the tax rate for MIT's tax identifier
+    mit_tax_identifier = models.ForeignKey(
+        TaxRate, on_delete=models.DO_NOTHING, blank=True, null=True
+    )
+
```

Comment:
Although we know we used separate fields to keep the data consistent w.r.t to the time of order, now that we are moving onwards, and added a foreign key to the same table as well. Do you think this might be the time we start using table versioning? Just like how we use product versions?

Reply:
Yeah, this is probably the better route, especially at the moment since we're not (AFAIK) using this actively quite yet. Doing this will be a bigger change so it will be a bit before it's done.

Reply:
I agree. Making a decision about doing this would probably depend upon:

1. What's the urgency of this?
2. Are we using the Tax feat in production yet? If yes, We'll also need to migrate the data.

FYI, @cachob

Reply:
I can't speak to urgency but we should not be collecting tax currently - the code is there but all the defined tax rates are turned off, which effectively disables the tax collection code.

Reply:
@jkachel Thanks for your response. My bad, My intention was actually to ask this from @cachob. 

@cachob Could you respond on https://github.com/mitodl/mitxpro/pull/2832#discussion_r1408939111?